modules/recon/contacts-credentials/hibp_paste.py

```diff
@@ -49,19 +52,22 @@ def module_run(self, accounts):
                     if paste['Source'] in sites:
                         fileurl = sites[paste['Source']].format(paste['Id'])
                         download = self.options['download']
-                    elif self.options['download'] == True:
+                    elif self.options['download']:
                         self.alert(f"Download not available for {paste['Source']} pastes.")
                     self.alert(f"{account} => Paste found! Seen in a {paste['Source']} on {paste['Date']} ({fileurl}).")
-                    if download == True:
-                        resp = self.request(fileurl)
-                        if resp.status_code == 200:
-                            filepath = f"{self.workspace}/{_safe_file_name(fileurl)}.txt"
-                            if not os.path.exists(filepath):
-                                dl = open(filepath, 'w')
-                                dl.write(resp.text.encode(resp.encoding) if resp.encoding else resp.text)
-                                dl.close()
-                            self.verbose(f"Paste stored at '{filepath}'.")
-                        else:
+                    if download:
+                        try:
```

Comment:
I'm not a fan of wrapping so much code in a try/except, even when the exception is specific. If you're only looking to catch failed requests (which it appears you are), then only include that line of code in the block. Make sense? Was there another reason you put so much code in the try block?

Reply:
Cleaned this one up now as well.